src/Network/Mattermost/Types.hs

```diff
@@ -1222,6 +1222,24 @@ instance A.ToJSON Preference where
     , "value"    .= preferenceValue
     ]
 
+data FavoriteChannelPreference =
+    FavoriteChannelPreference { favoriteChannelId :: ChannelId
+                           , favoriteChannelShow :: Bool
+                           } deriving (Read, Show, Eq)
+
+-- | Attempt to expose a 'Preference' as a 'FlaggedPost'
```

Comment:
This looks like an old comment. Can you update it to describe the new function?

Reply:
fixed in commit hash `ffd6d80`